### PR TITLE
adding custom favicon

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: config/sync/pineapple.settings.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+features:
+  node_user_picture: true
+  comment_user_picture: true
+  comment_user_verification: 1
+  favicon: 1
+logo:
+  use_default: 1
+favicon:
+  use_default: 0
+  path: 'public://Screenshot 2020-03-08 at 13.05.19.png'
```
Comment (ElXyfer) [inline config/sync/pineapple.settings.yml]: Rename this to an appropriate name please.

Comment (katlazarova) [reply config/sync/pineapple.settings.yml]: done

File: config/sync/pineapple.settings.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+features:
+  node_user_picture: true
+  comment_user_picture: true
+  comment_user_verification: 1
+  favicon: 1
+logo:
+  use_default: 1
+favicon:
+  use_default: 0
+  path: 'public://easy-recipe-delight-favicon.png'
```
Comment (ElXyfer) [inline config/sync/pineapple.settings.yml]: Favicons should be .ico not png, please get your favicon in the correct format

Comment (katlazarova) [reply config/sync/pineapple.settings.yml]: done

**Pull request closed without merging**